src/app/models/attribute-modifier.ts

```diff
@@ -0,0 +1,4 @@
+export class AttributeModifier {
```

Comment:
why do you need this for the navbar?

Reply:
Mocked hero hall uses heroes to populate dummy data by models, I included all, with full details.
Navbar only cares for hero.name and hero.id but the hero class required the equipments, and that required the modifiers.